### ISSUE TITLE
Top menubar icons not found

### ISSUE THREAD

**Issue by Bjorn248** [codegnv/business-web#196]
## Steps to Reproduce

1. Visit [http://bdev.c4gnv.com.s3-website-us-east-1.amazonaws.com/#/permit-locator](http://bdev.c4gnv.com.s3-website-us-east-1.amazonaws.com/#/permit-locator)
2. Note the missing icons in the top menu bar

## Expected Behavior

Icons should be visible

## Actual Behavior

<img width="982" alt="screenshot 2017-10-10 09 50 03" src="https://user-images.githubusercontent.com/1063707/31389942-857a26a2-ada0-11e7-98ce-af667730f4be.png">

**Comment by samanthawolfe** [comment]
@jgardner34 Would you like new version of the icons? This is something we could help with. Let me know the correct size needed.

**Comment by Bjorn248** [comment]
@samanthawolfe I think the icons are present in the repository, but they aren't getting loaded. We'll have to look into this no matter what but I do not believe the issue is lack of icons. It is likely a build issue.

**Comment by tillydray** [comment]
When doing `npm start`, I'm getting `webpack: Failed to compile.` preceded by a lot of errors. Could be that I'm doing something wrong, but I'm wondering if these issues have the same root cause (a build issue).

**Comment by ajcrites** [comment]
@JasonMFry not sure if this would make a difference, but can you try using `yarn` instead of `npm`? Also `rm -rf node_modules; yarn install` first.

**Comment by tillydray** [comment]
@ajcrites that fixed it, thanks! I thought I used `npm` last week and it worked, but maybe not.

**Comment by Bjorn248** [comment]
So, I took a gander at this and it seems that the SVGs here aren't being translated/transferred to the `assets` directory like the rest of the static assets seem to be. Also the `src` urls that are normally changed if webpack is working properly are not being changed. These two lines are of interest here: https://github.com/c4gnv/business-web/blob/develop/app/app.component.html#L26 and https://github.com/c4gnv/business-web/blob/develop/app/app.component.ts#L20, I also don't know if capitalization matters? However, with the rest of the static assets, such as https://github.com/c4gnv/business-web/blob/develop/app/components/businessType-list.component.html#L41 that src url gets translated/changed by webpack to point at `assets` so the urls referenced should probably be modified to include `../` at the beginning?

Just kind of throwing out what I've found so far.

Also, to replicate this issue locally, I did a `yarn build` followed by a cd into the `build` directory and then a `python -m SimpleHTTPServer 8000` to serve the static content in that directory. This only happens when the "prod" build is done, `yarn start` does not display this behavior for some reason.
